docs/RELEASE-NOTES.rst

```diff
@@ -1,11 +1,13 @@
 Release Notes for K8S BIG-IP Controller
 ============================================
 
-v1.1.0-dev
+v1.1.0
 ----------
 
 * Features
```

Comment:
Per @andrewjjenkins suggestion, can we reformat the sections for all version so that they have the following format:
**Added Functionality**
- feature 1
- feature 2

**Removed Functionality**
- any features that were removed (this should be empty for minor releases)

**Limitations**
- list of known limitations